### PR TITLE
[ADD] partner_design_mockup: AFC-2674 Added two drop down and search …

### PR DESCRIPTION
**Pull request by rupareliyahemangi145**
…button at top.

### CODE REVIEW

File: partner_design_mockup/__manifest__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     'name': 'Partner Design Mockup',
-    'version': '12.0.0.5',
+    'version': '12.0.0.6',
```
Comment (nils-vertel) [inline partner_design_mockup/__manifest__.py]: I would argue that this is an addition that enduser would see (adding dropdowns and a button) so the version number should be 12.0.0.1.0 atleast

File: partner_design_mockup/static/src/xml/partner_search.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                    <option value="perivouly_identified">Kand (tidigare identifierad)</option>
+                    <option value="identified_by_certifier">Identifierad genom intygsgivare</option>
+                </select>
+            </div>
+            <div class="search_button">
+                <button name="search_contact" class="btn btn-primary contact_search" string="Search">Search</button>
+            </div>
+        </t>
+    </t>
+</templates>
```
Comment (nils-vertel) [inline partner_design_mockup/static/src/xml/partner_search.xml]: Missing newline at end of file

File: partner_design_mockup/static/src/xml/partner_search.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<templates>
+    <t t-extend="ControlPanel">
+        <t t-jquery="ol.breadcrumb" t-operation="attributes">
+            <attribute name="style">width:20%</attribute>
+        </t>
+        <t t-jquery="ol.breadcrumb" t-operation="after">
+            <div class="search_place_view">
+                <select id="search_place" name="search_place" class="search_place_input">
+                    <option value="at_contact">vid kontakt</option>
+                    <option value="based_on_authority_needs">Utifran Myndighetens behov</option>
```
Comment (nils-vertel) [inline partner_design_mockup/static/src/xml/partner_search.xml]: Utifran -> Utifrån (If we allow swedish letters here)

**Review by nils-vertel — CHANGES_REQUESTED** [review]

File: partner_design_mockup/__manifest__.py (hunk 1 above)
Comment (rupareliyahemangi145) [reply partner_design_mockup/__manifest__.py]: Hi @nils-vertel I updated now.

File: partner_design_mockup/static/src/xml/partner_search.xml (hunk 2 above)
Comment (rupareliyahemangi145) [reply partner_design_mockup/static/src/xml/partner_search.xml]: Hi @nils-vertel I updated now.

File: partner_design_mockup/static/src/xml/partner_search.xml (hunk 3 above)
Comment (rupareliyahemangi145) [reply partner_design_mockup/static/src/xml/partner_search.xml]: Hi @nils-vertel I updated now.

**Pull request closed without merging**